FWCore/ParameterSet/python/Config.py

```diff
@@ -1034,10 +1053,10 @@ def __repr__(self):
     def __getattr__(self,attr):
         return self[attr]
 
-class SubProcess(_ConfigureComponent,_Unlabelable):
```

Comment:
@davidlange6 This change means you can't have in a configuration file 
```
   areco = cms.SubProcess(recoProcess)
```
and instead must always call ```addSubProcess``` from an existing ```Process```.